src/client/SplitPaneResizeContext.tsx

```diff
@@ -133,21 +147,39 @@ const splitPaneReducer = (
     case 'initialize':
       return {
         ...state,
-        sidebarWidth: initialSidebarWidth,
-        codeEditorWidth: initialCodeEditorWidth,
+        leftPanelWidth: initialLeftPanelWidth,
+        rightPanelWidth: initialRightPanelWidth,
         isInitialized: true,
       };
+    case 'resize':
```

Comment:
This change is unrelated to touch support. Best to add resize support in a separate PR.